packages/blocks/src/code-block/components/syntax-code-block.ts

```diff
@@ -102,15 +105,15 @@ class Syntax extends Module {
 
   constructor(quill: Quill, options: SyntaxCodeBlockOptions) {
     super(quill, options);
-    this.language = options.language;
+    this.language = options.lang;
```

Comment:
Thanks! Now it corresponds to  type of `SyntaxCodeBlockOptions`, I think https://github.com/toeverything/blocksuite/blob/d1b177466e5468a238f72ed0a1d38140af4b3f0f/packages/blocks/src/code-block/code-block.ts#L110 need to be changed to `lang` as well.